src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -344,6 +344,27 @@ public function testDumpOverridenGetters()
         $this->assertSame('baz', $r->invoke($baz));
     }
 
+    public function testDumpOverridenGettersWithConstructor()
+    {
+        $container = include self::$fixturesPath.'/containers/container34.php';
+        $container->compile();
+        $container->getDefinition('foo')
+            ->setOverriddenGetter('getInvalid', array(new Reference('bar', ContainerBuilder::IGNORE_ON_INVALID_REFERENCE)));
+        $dumper = new PhpDumper($container);
+
+        $dump = $dumper->dump(array('class' => 'Symfony_DI_PhpDumper_Test_Overriden_Getters_With_Constructor'));
+        $this->assertStringEqualsFile(self::$fixturesPath.'/php/services34.php', $dump);
+        $res = $container->getResources();
+        $this->assertSame(realpath(self::$fixturesPath.'/containers/container34.php'), array_pop($res)->getResource());
```

Comment:
You suggest that I should only check the presence of this resource in the array instead of what I'm doing here?